pkg/controllers/vdb/onlineupgrade_reconciler.go

```diff
@@ -101,6 +101,9 @@ func (o *OnlineUpgradeReconciler) Reconcile(ctx context.Context, _ *ctrl.Request
 		o.restartPrimaries,
 		// Handle restart of secondary subclusters
 		o.restartSecondaries,
+		// Reinstall default packages after all subclusters start after the upgrade
+		o.postNextStatusMsg,
+		o.installPackages,
```

Comment:
I'm thinking now that we need to do the install after we cleanup the transient. We don't have a way to know if the install is done or not. So, if we during cleanup of the transient we requeue, I think we will reinstall the packages again. Until we have a status condition to control the package install, I think the best bet is to move this after `o.deleteTransientSts`

Reply:
will do